tidb-cloud/pause-or-resume-tidb-cluster.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Pause or Resume a TiDB Cluster
+summary: Learn how to pause or resume a TiDB cluster.
+---
+
+# Pause or Resume a TiDB Cluster
+
+TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.
+
+Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).
+
+> **Note:**
+>
+> Currently, you cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). Once a Developer Tier cluster remains idle for 24 hours, the cluster hibernates automatically, and you can resume it by [connecting to your cluster](/tidb-cloud/connect-to-tidb-cluster.md) or clicking **Resume** for the cluster in the TiDB Cloud console.
+
+## Prerequisites
+
+- You cannot pause your cluster if it has any [Changefeeds](/tidb-cloud/changefeed-overview.md). You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster.
+- You can pause your cluster only when it is in the **Normal** state. If a cluster is in other states such as **Importing** or **Scaling**, you must wait for the current operation to be completed before pausing the cluster.
+- You cannot pause your cluster when it is backing up data, you can either wait for the current backup jobs to be completed or [delete the running backup job](/tidb-cloud/backup-and-restore.md#delete-a-running-backup-job).
+
+## Pause a TiDB cluster
```

Comment:
Will we add Open API method if API becomes public?

Reply:
If we want users to try API,  we can add a link here. It depends on when API is ready for trial.